docs/src/main/asciidoc/opentelemetry-metrics.adoc

```diff
@@ -139,17 +139,19 @@ To change any of the default property values, here is an example on how to confi
 ----
 quarkus.application.name=myservice // <1>
 quarkus.otel.metrics.enabled=true // <2>
-quarkus.otel.exporter.otlp.metrics.endpoint=http://localhost:4317 // <3>
-quarkus.otel.exporter.otlp.metrics.headers=authorization=Bearer my_secret // <4>
+quarkus.native.monitoring=jfr // <3>
```

Comment:
Is it something that should be enabled automatically if not manually set?

Adding @zakkak to the loop.

Reply:
> Is it something that should be enabled automatically if not manually set?

+1, if it's required it better be added by the quarkus extension, but there is no `builditem` to handle this at the moment AFAIK.

> Adding @zakkak to the loop.

Adding @roberttoyonaga as well :)

Reply:
I agree it must be handled automatically, if possible, but I'm not sure how.

Reply:
You will need to create a new build item. If you feel adventurous you can get some inspiration from https://github.com/quarkusio/quarkus/pull/44258, else I guess I can add it to my todo list :)

Reply:
https://github.com/quarkusio/quarkus/pull/21785 might be a cleaner example.

Reply:
Yes it should probably be automatic, otherwise users might get confusing problems at runtime if they forget.

Reply:
Remove the docs part and now JFR is activated when metrics are on